### PR TITLE
Ch 317: Allow access to explore-rate

### PR DESCRIPTION
**Pull request by eshtadc**
- Enable access to adjust explore rate
- Adjustments to control rate UI
- Convert control rate to initial intent (no more inversion)
- Set control rate default to .1
- Update tests

### CODE REVIEW

File: plugins/agent_types/AcquiaLiftAgent.inc [hunk 1]
```diff
@@ -223,12 +299,23 @@ class AcquiaLiftAgent extends PersonalizeAgentBase implements AcquiaLiftAgentInt
    */
   public function postSave($old_data) {
     $items = array();
-    $acquia_lift_control_rate = 0;
-    if (isset($this->data['control_rate'])) {
+    $acquia_lift_control_rate = .1;
+    $acquia_lift_explore_rate = .2;
+    dpm($this->data, 'postSave data');
```
Comment (katbailey) [inline plugins/agent_types/AcquiaLiftAgent.inc:304]: Oops :)

File: plugins/agent_types/AcquiaLiftAgent.inc [hunk 2]
```diff
@@ -316,7 +404,7 @@ class AcquiaLiftAgent extends PersonalizeAgentBase implements AcquiaLiftAgentInt
     );
     // We only show the lift over default if we are using a non-zero percentage of
     // users as a control group.
-    $show_default = isset($this->data['control_rate']) && (int) $this->data['control_rate'] > 0;
+    $show_default = isset($this->data['control']['control_rate']) && (int) $this->data['control']['control_rate'] > 0;
```
Comment (katbailey) [inline plugins/agent_types/AcquiaLiftAgent.inc:407]: Just curious why we're now nesting the control rate one level deeper inside the data array?

File: plugins/agent_types/AcquiaLiftAgent.inc (hunk 1 above)
Comment (eshtadc) [inline plugins/agent_types/AcquiaLiftAgent.inc:304]: Darn it.

File: plugins/agent_types/AcquiaLiftAgent.inc (hunk 2 above)
Comment (eshtadc) [inline plugins/agent_types/AcquiaLiftAgent.inc:407]: It had to do with the nested structure necessitated for the collapsing stuff.  I don't like it nested deeper though.  How can I keep the data structure the same?  It seems it copies directly from the form.

**Pull request closed without merging**